packages/config/src/ios/utils/Xcodeproj.ts

```diff
@@ -58,21 +61,73 @@ export function getPbxproj(projectRoot: string) {
   return project;
 }
 
-export function removeComments([item]: any[]): boolean {
-  return !item.endsWith(`_comment`);
+export type ProjectSection = { [key: string]: ProjectSectionItem };
+export type ProjectSectionItem = {
+  isa: string;
+  attributes: {
+    TargetAttributes: {
+      [key: string]: {
+        DevelopmentTeam?: string;
+        ProvisioningStyle?: string;
+      };
+    };
```

Comment:
Maybe this is cleaner
```suggestion
    TargetAttributes: Record<
      string,
      {
        DevelopmentTeam?: string;
        ProvisioningStyle?: string;
      }
    >;
```